Makefile

```diff
@@ -508,8 +516,7 @@ security-test:
 .PHONY: common-test
 common-test:
 	go test ${T} ./pkg/...
-	# Execute bash shell unit tests scripts
-	./tests/scripts/scripts_test.sh
```

Comment:
Don't delete.

Reply:
This script is no longer needed, because it only tests the functions related to IP parsing. IP parsing in go is a part of the standard library.